presto-product-tests/src/main/java/io/prestosql/tests/kafka/KafkaAvroSmokeTest.java

```diff
@@ -107,7 +107,9 @@ private static Requirement createAvroTable(String schemaPath, String tableName,
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
         GenericData.Record record = new GenericData.Record(schema);
         values.forEach(record::put);
-        try (DataFileWriter<GenericRecord> dataFileWriter = new DataFileWriter<>(new GenericDatumWriter<>(schema))) {
+        try {
```

Comment:
These lines shouldn't be changed in this commit. (looks like a rebase error)

Reply:
Something looks off indeed